boards/vcu118/pcie/synth/firmware/cfg/vcu118_infra_pcie.dep

```diff
@@ -32,7 +32,6 @@
 
 
 src vcu118_infra_pcie.vhd
-src -c components/ipbus_util clocks/clocks_usp_serdes.vhd ipbus_clock_div.vhd led_stretcher.vhd
```

Comment:
With this change, the CI builds failed since the `clocks_usp_serdes` entity wasn't included anymore. Re-adding this line without `led_stretcher.vhd` resolved this for me - see https://github.com/ipbus/ipbus-firmware/commit/91701f61ebd0d12917e53efe87a834c4862acd3a

If you saw different behaviour when testing locally, we should try to confirm why. Otherwise, could you add the changes for both depfiles from https://github.com/ipbus/ipbus-firmware/commit/91701f61ebd0d12917e53efe87a834c4862acd3a to your branch?